guide/mechanics/damageFormula.md

```diff
@@ -8,27 +8,35 @@ sidebar_label: Damage Formula
+**SumDamageModifiers:** If you’re looking at a weapon with 75% weapon damage and no other modifiers, this number is 0.75. However, this should be the sum of every applicable damage modifier. Consult the [Damage Types](mechanics/damageTypes.md) guide for more information about how to calculate this.
 
-**C:** Target Crit Multiplier For anything that cannot crit, this value is 1. Each weakpoint on an enemy has it's on crit multiplier. For most enemies, this value is 0.5. Some enemies such as scars, outlaw shotgunners, and Ash Titan's have higher or lower values.
+**TypeModifier:** This represents the damage bonus (or penalty) applied based on the damage type of your attack and the health type of the enemy. All attacks fall into one of 5 categories: Impact, Acid, Electric, Ice, or Fire. A table showing what the value of D should be is shown in the table below
 
-**D:** Critical Damage Modifiers.  If your item cannot crit, this value is 0. If your item can crit, this value is the total of all applicable crit damage modifiers. If you have 10% on your primary gun (gear icon) and 50% on your secondary gun (gear icon), these values are 0.1 and 0.5, respectively. However, some slots other than weapons can roll javelin-wide (javelin icon) critical inscriptions. These inscriptions should be included.
+**SumDebuffModifiers:** Sum of all debuffs applied to an enemy that cause them to take additional damage. Currently Acid, Interceptor's Target Beacon, and Colossus' Taunt. Beacon and Taunt do not stack with each other. Thus, the maximum value is 
 
-**E:** Damage type multiplier. This represents the damage bonus (or penalty) applied based on the damage type of your attack and the health type of the enemy. All attacks fall into one of 5 categories: Impact, Acid, Electric, Ice, or Fire. A table showing what the value of D should be is shown in the table below
+**ItemCritBonus:** For anything that cannot crit, this value is 1. For anything that can crit, this value depends on the damage source. Every weapon and ability that can crit has a crit multiplier specific to the item. Blastback is 2.4, Plasma Star is 1.5. a full list of datamined values can be found on [AnthemArchive](http:AnthemArchive.com) Please note that shielded enemies cannot be critically hit for damage purposes.
 
-**F:** Debuff multiplier. If the enemy is neither afflicted by Acid nor Target Beacon, this value is 1. This value increased by 0.25 if the enemy is afflicted by Acid, and increases by 0.33 if the enemy is marked with Target Beacon. If both are applied, this value is 1.58.
+**WeakPointMultiplier:** Target Crit Multiplier For anything that cannot crit, this value is 1. Each weakpoint on an enemy has it's on crit multiplier. For most enemies, this value is 0.5. Some enemies such as scars, outlaw shotgunners, and Ash Titan's have higher or lower values.
```

Comment:
I would lead with "for common enemies such as scar scrappers and outlaw gunners, this is 0.75. For more advanced enemies, it is 0.5. It is rare for anything to take 'full' damage from crits, but some boss body parts are extra vulnerable."

Reply:
and I would lead *that* with "Weak point multiplier represents an enemy's critical hit resistance, and is usually less than 1. Most enemies have 25-50% crit resistance" or something

Reply:
Maybe call this `WeakPointResistanceMultiplier'